### ISSUE TITLE
Light Mode | 'Sort' icons wrong color

### ISSUE THREAD

**Issue by SamusElderg** [spartan-protocol/SpartanProtocol-DAppV2#782]
Just a visual minor detail, the `sort` icon handling (manipulate SVG with classes and states) works good for dark mode, but haven't adjusted the 'white' state to suit light mode. I think 'black' would be best.

This is an easy one for a first timer getting their toes wet, this is the code required to fix:
```
.orderDesc .down {
    fill: white;
}
  
.orderAsc .up {
    fill: white;
}
```

Which needs to be added to:
`src\assets\styles\global.scss`
and nested inside:
`body.dark-theme {}`
(near the bottom of the file)

and then we need to change the existing rules (the ones that are NOT nested in the dark-theme rule):
```
.orderDesc .down {
    fill: black;
}
  
.orderAsc .up {
    fill: black;
}
```

**Comment by andyosei12** [comment]
I'm willing to work on this

**Comment by Sir-Rumeh** [comment]
Assign me for this

**Comment by SamusElderg** [comment]
> Assign me for this

@andyosei12 has already put hand up to do it, but will reach out if it doesn't happen. In the meantime, if there are any other suggestions you may have for changes in the DApp that you would like to code in, you can propose an issue and/or PR

**Comment by SamusElderg** [comment]
closed as of 56d8808967af5ca4f61c575375a9576238922f89

**Comment by Sir-Rumeh** [comment]
Alright. I will keep you updated on anything I decide to work on
